pydal/_gae.py

```diff
@@ -83,3 +78,6 @@ def _validate(self, value):
                 return decimal.Decimal(value)
             raise TypeError("Property %s must be a Decimal or string."
                             % self._name)
```

Comment:
The problem is that if the first line succeeds (we have GAE) and one of the
latter fails because we have a bug in those lines, or a missing module,
etc, it will think we are not on GAE and we do not get a traceback


On Fri, Mar 13, 2015 at 11:20 AM, Giovanni Barillari <
notifications@github.com> wrote:

> In pydal/_gae.py
> <https://github.com/web2py/pydal/pull/82#discussion_r26399464>:
>
> > @@ -83,3 +78,6 @@ def _validate(self, value):
> >                  return decimal.Decimal(value)
> >              raise TypeError("Property %s must be a Decimal or string."
> >                              % self._name)
>
> And why is that a problem? If the first line of the try block fails the
> interpreter will stop the execution and jump to the except one, no matter
> of how many lines you have inside the block..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/pydal/pull/82/files#r26399464>.
>